### PR TITLE
added courses-crud, created routes for courses

### PR DESCRIPTION
**Pull request by danraponga**
*created courses schemas

*created courses-crud

*created courses routes that are included in api/routers

*removed items related code 

*fixed a bug in relationships between course and user tables

### CODE REVIEW

File: app/api/courses/routes.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+router = APIRouter()
+
+
+@router.post('/')
+def read_courses(db: Session = Depends(get_db), skip: int = 0, limit: int = 100):
+    courses = crud_course.get_multi(db, skip=skip, limit=limit)
+    return courses
+
+
+@router.post('/create')
```
Comment (vladysllav) [inline app/api/courses/routes.py]: Endpoint should be "courses/" POST

File: app/api/courses/routes.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+def create_course(*, db: Session = Depends(get_db), course_in: CourseCreate,
+                  current_user: User = Depends(get_current_user)):
+    course = crud_course.create(db, obj_in=course_in, current_user=current_user)
+    
+    return course
+
+
+@router.post('/update/{course_id}')
```
Comment (vladysllav) [inline app/api/courses/routes.py]: Endpoint should be "courses/{id}" PUT or PATCH

File: app/schemas/course.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+from pydantic import BaseModel
+
+
+class BaseCourse(BaseModel):
+    name: Optional[str] = Field(None, example='ULTIMATE Python course')
+    description: Optional[str] = Field(None, example='The best Python course in the World!')
+    is_active: Optional[bool] = Field(True, example=True)
+
+
+class CourseRead(BaseCourse):
```
Comment (vladysllav) [inline app/schemas/course.py:14]: We should have id field as well

**Review by vladysllav — CHANGES_REQUESTED** [review]

File: app/api/courses/routes.py (hunk 1 above)
Comment (danraponga) [reply app/api/courses/routes.py]: fixed

File: app/api/courses/routes.py (hunk 2 above)
Comment (danraponga) [reply app/api/courses/routes.py]: fixed

File: app/schemas/course.py (hunk 3 above)
Comment (danraponga) [reply app/schemas/course.py:14]: fixed

**Review by vladysllav — APPROVED** [review]

**Pull request merged**